src/libs/ajax.js

```diff
@@ -1122,8 +1122,9 @@ const Apps = signal => ({
   app: (project, name) => {
     const root = `api/google/v1/apps/${project}/${name}`
     return {
-      delete: () => {
-        return fetchLeo(root, _.mergeAll([authOpts(), { signal, method: 'DELETE' }, appIdentifier]))
+      delete: deleteDisk => {
+        return fetchLeo(`${root}${qs.stringify({ deleteDisk }, { addQueryPrefix: true })}`,
```

Comment:
I don't see this option in the swagger, is it actually available?

Reply:
Yes, there's discussion on slack, they missed adding it to swagger.